cypress/e2e/forside.cy.ts

```diff
@@ -0,0 +1,30 @@
+describe('Forside tester', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+  it('Tekst laster fra Sanity', () => {
+    cy.get(`[data-testid='forsideTittel']`).contains('Endringsmelding');
+  });
+  it('Språkvelger endrer språk', () => {
+    cy.get(`[data-testid='språkvelger']`).select('en');
+    cy.get(`[data-testid='forsideTittel']`).contains('Change message');
+  });
+  it('Fornavn hentet fra backend', () => {
+    //Må endre til det fornavnet vi vil hente, mocket
+    cy.get(`[data-testid='hilsenFornavn']`).contains('Hei, Fornavn!');
+  });
+  it('Kan ikke gå videre før samtykkepanel er bekreftet', () => {
+    cy.get(`[data-testid='startKnapp']`).click();
+    //Kan feilmeldingsteksten kanskje hentes på en annen måte eller sjekkes på en annen måte sånn at testen funker selvom teksten blir endret
+    cy.get(`[data-testid='samtykkepanelFeilmelding']`).contains(
+      'Du må samtykke før du kan fortsette',
+    );
+  });
+  it('Samtykkepanel bekreftet før man kan gå videre', () => {
+    cy.get(`[data-testid='samtykkepanel']`).check();
+    cy.get(`[data-testid='startKnapp']`).click();
+  });
```

Comment:
Usikker på hvordan testene henger sammen i cypress. Er det sånn at denne nå tester som man har fått feilmelding tidligere eller som om man trykker for første gang? 

Det bør det kanskje være en test som både sjekker at du får lov til å trykke neste om du har huket av, en hvor du ikke får lov om du ikke har huket av og en hvor du ikke har huket av og så får gå videre om du huker av etterpå (når feilmelding er vist)

Reply:
https://github.com/bekk/nav-familie-endringsmelding/pull/63/commits/739fa5a85f1fa049a2d5ca5cc7fee5b4134b833b